xmlschema/locale/ru/LC_MESSAGES/xmlschema.po

```diff
@@ -0,0 +1,94 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
I will add copyright info on this part.
If it's not a problem also add author info.